package.json

```diff
@@ -38,6 +38,7 @@
     "convict": "^4.0.1",
     "cors": "^2.8.4",
     "express": "^4.16.2",
+    "jest-worker": "^21.3.0-beta.14",
```

Comment:
22.0.0 is coming any day now, will hold on merging until then

Reply:
whats the difference?

Reply:
No code difference, I just dislike beta dependencies

Reply:
There will be a code difference, actually: https://github.com/facebook/jest/pull/5068